xclim/sdba/base.py

```diff
@@ -29,12 +29,21 @@ def __init__(self, **kwargs):
             setattr(self, key, val)
             self._parameter_names.append(key)
 
+    def _add_parameter(self, name, value):
+        setattr(self, name, value)
+        self._parameter_names.append(name)
+
     @property
     def parameters(self):
         """All parameters as a dictionary."""
         return {key: getattr(self, key) for key in self._parameter_names}
 
     def __str__(self):
+        return (
+            f"<{self.__class__.__name__}({json.dumps(self.parameters, default=str)})>"
+        )
+
+    def __repr__(self):
         return (
```

Comment:
Why not use repr to return a string that could be "eval"led  to return the actual object ?

Reply:
Well, that's what `__str__` does? Also, I kept the old output, but there is no need really. We could have only one of both.